templates/skeleton/app/routes/account/login.tsx

```diff
@@ -1,12 +1,10 @@
 import {
-  type MetaFunction,
   type ActionFunction,
   type LoaderArgs,
+  type ErrorBoundaryComponent,
   redirect,
-  json,
 } from '@shopify/remix-oxygen';
```

Comment:
Just spitballing here: Would it be too outré to separate type imports from function imports, even when they come from the same package? I find this easier to quickly scan and interpret:

```ts
import type { ActionFunction, LoaderArgs, ErrorBoundaryComponent } from '@shopify/remix-oxygen'

import { redirect } from '@shopify/remix-oxygen'

```

Reply:
I don't think I've ever seen it done before; seems a little weird. Many people also rely on VSCode to auto-import something when they use it, and I think VSCode may end up mixing things in the end anyway?